### PR TITLE
 Update tc-functions

### PR DESCRIPTION
**Pull request by CardealRusso**

### CODE REVIEW

File: etc/init.d/tc-functions
Comment (CardealRusso) [inline etc/init.d/tc-functions:1]: grep itself returns errorlevel

**Comment by clbr** [comment]
I've pulled the first patch, thanks. I'm not sure about the others, as using grep there instead of shell builtins will be slower. Is there some other benefit than just a cleanup?

**Comment by CardealRusso** [comment]
> I've pulled the first patch, thanks. I'm not sure about the others, as using grep there instead of shell builtins will be slower. Is there some other benefit than just a cleanup?
Yes, I wondered the same thing. I initially thought that there wouldn't be a humanly perceptible change in slowness?

**Comment by clbr** [comment]
There will be on older hw (486, 586).

**Pull request closed without merging**